File: test_int_array.cpp
// Be sure to compile with g++ not gcc
#include "int_array.h"

int main()
{
  int_array myv;
  srand(time(0));
  for (int i = 0; i < 55; i++)
    myv.push(rand() % 100);
  cout << "int array UNSORTED" << endl;
  cout << myv;
  myv.sort();
  cout << "int array SORTED" << endl;
  cout << myv;
  return 0;
}

File: test_int_array.java
import java.util.Random;

class test_int_array
{
  public static void main(String[] args)
  {
    int_array myv = new int_array();
    Random rand = new Random();
    for (int i = 0; i < 55; i++)
      myv.push(rand.nextInt(100));
    System.out.println("Int array UNSORTED:");
    myv.print();
    myv.sort();
    System.out.println("Int array SORTED:");
    myv.print();
  }
}
